packages/jest-core/src/runJest.ts

```diff
@@ -200,6 +200,7 @@ export default async function runJest({
     allTests = await sequencer.shard(allTests, globalConfig.shard);
   }
 
+  // jhwang allTests actually means all test suites
```

Comment:
?

Reply:
Oh whoops. Will delete, just notes I was taking to myself.